WIP -Playbook merger

geoand opened the pull request.

geoand commented:
This will have series of commits. 
I will remove the `WIP` designation once I'm done in order to indicate that it's ready for review/merge

cmoulliard commented on ansible/playbook/roles/generate_inventory/templates/cloud.inventory.j2:
```diff
@@ -1,56 +1,58 @@
 openshift_pkg_version="-3.7.0"
+
 openshift_deployment_type=origin
-openshift_enable_service_catalog=true
```
Is there a reason why you removed it as it is required for the lab ? should we enable/disable base on a property passed when we generate the inventory ? @geoand

geoand replied:
Actually I haven't removed it, I just changed it's position in the file to match where it is in the playbook that exists under `virtualbox`. See this: https://github.com/snowdrop/cloud-native-infra/blob/365b049f557ee32f6350aac53ed8e7eb6009dbdc/ansible/playbook/roles/generate_inventory/templates/cloud.inventory.j2#L36

cmoulliard commented:
Do you want that I merge it ?

geoand commented:
Not yet, I want to add some more commit as well, so we can review them as a whole :)

geoand commented:
@cmoulliard This should be ready for review.
I removed all Ansible related functionality from the `virtualbox` directory and updated the playbooks in the `ansible` to reflect your latest changes.

geoand commented:
I have updated the PR to consolidate the latest changes from master
